nginx-redirects.conf

```diff
@@ -334,10 +334,18 @@ server {
 
 server {
     include /etc/nginx/ssl.default.conf;
+    include /etc/nginx/api-proxy.conf;
     server_name chimpandsee.org www.chimpandsee.org;
     return 301 https://www.zooniverse.org/projects/sassydumbledore/chimp-and-see;
```

Comment:
This line will need to be wrapped in a `location /` block to stop it taking precedence over the API proxy `location`s. At the moment it's still returning a redirect:

```
$ curl -v -H "Host: www.chimpandsee.org" localhost:8080/_ouroboros_api/
...
< HTTP/1.1 301 Moved Permanently
...
< Location: https://www.zooniverse.org/projects/sassydumbledore/chimp-and-see
```